src/apps/backend/modules/account/internal/account-reader.ts

```diff
@@ -53,15 +56,64 @@ export default class AccountReader {
     return AccountUtil.convertAccountDBToAccount(accDb);
   }
 
+  public static async getAccountByPhoneNumber(
```

Comment:
Can you refactor this method to use `getAccountByPhoneNumberOptional`, and if that returns null, throw an exception? This will keep code DRY